testdata/TestPointers__array.golden

```diff
@@ -1 +1 @@
-&[1]float32{1}
\ No newline at end of file
+&[1]float32{float32(1)}
```

Comment:
This, for example, is an update I wouldn't expect to see: `[1]float32{1}` is completely valid and so the type qualifier `float32(1)` shouldn't be needed.